projects/create-a-project.md

```diff
@@ -1,10 +1,40 @@
-# Create a Project
+# Create a Gradientº Project
 
 ## About
 
-A project is simply a directory or name for a set of related files and jobs. 
+As of 7.0, a Gradientº Project that represents a container for running Experiments.
 
-## Initializing a New Project
+They can be powered by GradientCI (and GitHub), or alternately they can be "Standalone".
```

Comment:
```suggestion
Projects can be powered by GradientCI (and GitHub), or alternatively they can be "Standalone".
```

Reply:
hah, `alternately` and `alternatively` are apparently the same in North America, according to google ;)